extension/ts/services/renderFormObserver.ts

```diff
@@ -1,4 +1,4 @@
-import {FORM_RENDER_EVENT} from "../constants";
+const FORM_RENDER_EVENT = "library-thing-form-rendered";
```

Comment:
So now if a component needs the `FORM_RENDER_EVENT`, importing this will automatically start the observer service.

This will work even if we split into different files again